cves/CVE-2004-0751.yml

```diff
@@ -94,11 +102,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: |
+    Found using a RewriteRule to do reverse proxying to an SSL server running IIS.
+    To reproduce, certain directives are required in httpd.conf. 
+  date: 2004-07-15
+  automated: 
+  google: false
+  contest: true
```

Comment:
Do you have information on the contest that this was found in?

Reply:
I misinterpreted the instructions as it was either discovered by a Google employee or it was in a contest. I will make that change.